### ISSUE TITLE
`TypeError: testCases is not iterable`

### ISSUE THREAD

**Issue by ZanzyTHEbar** [promptfoo/promptfoo#1040]
My `promptfooconfig.yml`:
```yml
description: "verify"

prompts:
  - file://verify.prompt.txt
providers:
  - file://verify.provider.yml
tests: tests/*.test.yml
```
Test file:
```yml
  description: "Validation of the code changes"
  vars:
    preBuildState: file://assets/shared/pre_build.go
    changes: file://assets/valid/changes.md
    postBuildState: file://assets/valid/post_build.go
    diffs: file://assets/valid/diff.txt
  assert:
    - type: is-json
    - type: is-valid-openai-tools-call
    - type: javascript
      value: |
        var args = JSON.parse(output[0].function.arguments)
        return !(
          args.hasSyntaxErrors ||
          args.hasRemovedCodeErrors ||
          args.hasDuplicationErrors ||
          args.hasReferenceErrors            
        )

```

Previous to my update this worked fine. I had all of my tests in my `/tests` folder. After update - this no longer works.

```
Cache is disabled.
Loaded provider openai:gpt-4o from verify.provider.yml
/home/daofficialwizard/.local/share/pnpm/global/5/.pnpm/promptfoo@0.67.0_@aws-sdk+client-bedrock-runtime@3.598.0_@azure+identity@4.2.1_@azure+openai-_o4xgjjjr6zw7j63g3trvuro4vm/node_modules/promptfoo/dist/src/testCases.js:177
                for (const testCase of testCases) {
                                       ^

TypeError: testCases is not iterable
    at loadTestsFromGlob (/home/daofficialwizard/.local/share/pnpm/global/5/.pnpm/promptfoo@0.67.0_@aws-sdk+client-bedrock-runtime@3.598.0_@azure+identity@4.2.1_@azure+openai-_o4xgjjjr6zw7j63g3trvuro4vm/node_modules/promptfoo/dist/src/testCases.js:177:40)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async resolveConfigs (/home/daofficialwizard/.local/share/pnpm/global/5/.pnpm/promptfoo@0.67.0_@aws-sdk+client-bedrock-runtime@3.598.0_@azure+identity@4.2.1_@azure+openai-_o4xgjjjr6zw7j63g3trvuro4vm/node_modules/promptfoo/dist/src/main.js:134:25)
    at async runEvaluation (/home/daofficialwizard/.local/share/pnpm/global/5/.pnpm/promptfoo@0.67.0_@aws-sdk+client-bedrock-runtime@3.598.0_@azure+identity@4.2.1_@azure+openai-_o4xgjjjr6zw7j63g3trvuro4vm/node_modules/promptfoo/dist/src/main.js:545:48)
    at async Command.<anonymous> (/home/daofficialwizard/.local/share/pnpm/global/5/.pnpm/promptfoo@0.67.0_@aws-sdk+client-bedrock-runtime@3.598.0_@azure+identity@4.2.1_@azure+openai-_o4xgjjjr6zw7j63g3trvuro4vm/node_modules/promptfoo/dist/src/main.js:704:9)

Node.js v22.3.0
```

That is the error.

**Comment by ZanzyTHEbar** [comment]
Seems the issue is actually a missing `-` at the top of the test file.

**Comment by mldangelo** [comment]
We deployed a fix for this in [0.68.0](https://github.com/promptfoo/promptfoo/releases/tag/0.68.0) as well. Thanks for reporting the issue and helping make promptfoo better @ZanzyTHEbar!
